controllers/booking.js

```diff
@@ -1,7 +1,40 @@
 
-// Show new booking form.
-exports.booking = (req, res) => {
-    const name = req.params.id;
+module.exports = {
+    // Show the Booking page
+    booking_get: (req, res) => {
+        res.render('booking.njk', {title: 'Booking Form'});
+    },
 
-    res.render('booking.njk', {title: 'Booking Form'});
-};
\ No newline at end of file
+    // Process and validate the booking page
+    bookingProcess_post: async (req, res) => {
```

Comment:
Is there a reason we can't underscore all of this? (`booking_process_post`). No big deal, it just bugs me as a style thing.